roxie/ccd/ccdmain.cpp

```diff
@@ -770,10 +770,15 @@ int STARTQUERY_API start_query(int argc, const char *argv[])
             removeLog();
         if (topology->hasProp("@logfile"))
         {
+#ifndef _CONTAINERIZED
             Owned<IComponentLogFileCreator> lf = createComponentLogFileCreator(topology, "roxie");
             lf->setMaxDetail(TopDetail);
             lf->beginLogging();
             logDirectory.set(lf->queryLogDir());
+#else
+            updateContainerizedLogMsgHandler(getCategoryLogMsgFilter(MSGAUD_all, MSGCLS_all, TopDetail));
```

Comment:
I think should look like:
```
    installContainerLogHandler();
```

and logFields and detail level - should be attributes in defaultYaml (see line 527).

in installContainerLogHandler() it should look at the config (queryComponentConfig()) and pick up properties it needs, but probably should also default to something generic if missing.